src/main/java/ball/characteristics/Ball.java

```diff
@@ -0,0 +1,27 @@
+package ball.characteristics;
+
+public class Ball {
+    private int number;
+    private String color;
```

Comment:
The color field should be of the enum type Color, not String, to enforce the use of predefined color values.